homeassistant/components/workday/__init__.py

```diff
@@ -17,6 +18,13 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     country: str | None = entry.options.get(CONF_COUNTRY)
     province: str | None = entry.options.get(CONF_PROVINCE)
 
+    if country and CONF_LANGUAGE not in entry.options:
+        cls: HolidayBase = country_holidays(country, subdiv=province)
+        default_language = cls.default_language
```

Comment:
We should get the default language from Home Assistant.

Reply:
https://github.com/home-assistant/core/pull/104472